src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -52,36 +52,31 @@ export function transform(formConfig, form) {
   });
 }
 
+export function isValidDisability(disability) {
+  const invalidDisabilityError = (error => /^instance.disabilities\[/.test(error.property));
+  const v = new Validator();
+  const result = v.validate(
+    { disabilities: [disability] },
+    fullSchemaIncrease
+  );
 
-export const isPrefillDataComplete = (formData) => {
-  const { socialSecurityNumber, vaFileNumber, gender,
-    dateOfBirth, servicePeriods } = formData;
-  const first = get('fullName.first', formData);
-  const last = get('fullName.last', formData);
-  const country = get('veteran.mailingAddress.country', formData);
-  const addressLine1 = get('veteran.mailingAddress.addressLine1', formData);
-  const emailAddress = get('veteran.emailAddress', formData);
-  const primaryPhone = get('veteran.primaryPhone', formData);
-  const accountType = get('directDeposit.accountType', formData);
-  const routingNumber = get('directDeposit.routingNumber', formData);
-  const bankName = get('directDeposit.bankName', formData);
-  const noBank = get('directDeposit.noBank', formData);
-  const hasVeteranDetails = first && last && gender && dateOfBirth && (socialSecurityNumber || vaFileNumber);
-  const hasPrimaryAddressInfo = country && addressLine1 && emailAddress && primaryPhone;
-  const hasPaymentInfo = noBank || (accountType && routingNumber && bankName);
-  const hasMilitaryHistoryInfo = servicePeriods && servicePeriods.length > 0;
-  return !!(hasVeteranDetails && hasPrimaryAddressInfo && hasPaymentInfo && hasMilitaryHistoryInfo);
-};
+  if (result.errors.find(invalidDisabilityError)) {
+    Raven.captureMessage(`Invalid-disability-prefilled: ${disability}`);
+    return false;
+  }
+  return true;
+}
 
+export function filterInvalidDisabilities(disabilities) {
+  return disabilities.filter(disability => isValidDisability(disability));
+}
 
-export function prefillTransformer(pages, formData, metadata, state) {
-  let newData = formData;
-  const isPrefilled = state.prefilStatus === PREFILL_STATUSES.success;
-  const hasRequiredInformation = isPrefillDataComplete(formData);
+export function transformDisabilities(disabilities) {
+  return disabilities.map(disability => set('disabilityActionType', 'INCREASE', disability));
+}
 
-  if (isPrefilled && hasRequiredInformation) {
-    newData = set('prefilled', true, newData);
-  }
+export function prefillTransformer(pages, formData, metadata) {
+  const newData = set('disabilities', filterInvalidDisabilities(transformDisabilities(formData.disabilities)), formData);
```

Comment:
We could compare the length of the new and original data if we wanted to store that information in a metadata flag to determine whether to display a message on the rated disabilities page (question asked [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/11202#issuecomment-399539628)).